src/molecules/Modal/Modal.native.js

```diff
@@ -0,0 +1,246 @@
+import React from 'react';
+import RNModal from 'react-native-modal';
+import styled from 'styled-components/native';
+import PropTypes from 'prop-types';
+import Space from '../../atoms/Space';
+import Flex from '../../atoms/Flex';
+import Size from '../../atoms/Size';
+import View from '../../atoms/View';
+import Button from '../../atoms/Button';
+
+const CenteredModalBody = styled(View)`
+  background-color: ${(props) => props.theme.colors.background[100]};
+  border-radius: ${(props) => props.theme.spacings.xsmall};
+`;
+
+const FullscreenModalBody = styled(View)`
+  background-color: ${(props) => props.theme.colors.background[400]};
+`;
+
+const BottomsheetModalBody = styled(View)`
```

Comment:
why can't this be a ModalContentContainer and then get the styles basis the type? The way it's done here you'll end up creating 3 views in the memory but use just one at a time.